### ISSUE TITLE
No key deletion support.

### ISSUE THREAD

**Issue by sshagy** [ionelmc/django-redisboard#45]
Hi, everyone!

I would like to be able to delete both a specific key and clear the entire database. True, the latter requires special permissions to execute commands: **FLUSHDB/FLUSHALL**. But key deletion is available by default: **DEL key**

Does this package fit this functionality?

**Comment by ionelmc** [comment]
You could enumerate the keys yourself if you have restrictions placed in. The library assumes the more efficient flush commands are available.

**Comment by sshagy** [comment]
That's right, I have no problem deleting keys from the console. I mean, it would be great if it was possible to delete keys through the django admin panel (add a delete button for each key, or make checkboxes and one delete button). Does it make sense to add this to this package in "Inspect RedisServer" section?

![изображение](https://user-images.githubusercontent.com/35382350/93877060-81df7280-fce0-11ea-894d-38b77e864cf1.png)

From time to time there remains garbage with TTL -1, which has to be cleaned by hand through the console. Perhaps there are people who will understand me. I'm interested in the opinion of the author of this package :)

**Comment by ionelmc** [comment]
Ooooof. Looks like I confused the bug report https://github.com/ionelmc/python-redis-lock. 

Anyway, yes, there could be a delete button. Are you interested in making a PR?

**Comment by sshagy** [comment]
Lol, anything can happen)

Yes, I'm interested. Although I had little practice in customizing django-admin panels, I will try to do it in my free time )
